docs/add_a_validation.md

```diff
@@ -0,0 +1,295 @@
+# Add a validation
+
+We want to add the option to configure list of authors from which the tweets are allowed. To achieve this, we have to:
```

Comment:
We want to add the option to configure **a** list of authors **whose** tweets are allowed.